README.md

```diff
@@ -62,35 +75,39 @@ individual files in /usr/share/doc/*/copyright.
 Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
 applicable law.
 
-root@buildcowsay> image-builder-configure
+root@buildcowsay:~# image-builder-configure
 Login (cloud.scaleway.com):                     # yourmail
 Password:                                       # yourpassword
-root@buildcowsay> mkdir vivid-cowsay
-root@buildcowsay> cp Makefile vivid-cowsay
-root@buildcowsay> cd vivid-cowsay
-root@buildcowsay> touch Dockerfile
-root@buildcowsay> ls -l
+root@buildcowsay:~# mkdir vivid-cowsay
+root@buildcowsay:~# cp Makefile vivid-cowsay
+root@buildcowsay:~# cd vivid-cowsay
+root@buildcowsay:~/vivid-cowsay# touch Dockerfile
```

Comment:
I think you should rename Makefile to Makefile.example, and add a standard Dockerfile.example, then here, you can just put 

`cp Dockerfile.example vivid-cowsay/Dockerfile`, same for `Makefile`